### PR TITLE
24: [FEATURE] Add Expense and ExpenseCategory entities

### PR DESCRIPTION
**Pull request by eIIka**
- added Expense and ExpenseCategory models
- added corresponding DTO classes

## Checklist before requesting a review
- [X] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

### CODE REVIEW

File: hillel-expense-tracker-common/src/main/java/ua/ithillel/expensetracker/dto/ExpenseCategoryDTO.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package ua.ithillel.expensetracker.dto;
+
+public class ExpenseCategoryDTO {
+}
```
Comment (ownerofglory) [inline hillel-expense-tracker-common/src/main/java/ua/ithillel/expensetracker/dto/ExpenseCategoryDTO.java:14]: class members are missing, perhaps you've forgotten to commit the changes?

File: hillel-expense-tracker-common/src/main/java/ua/ithillel/expensetracker/dto/ExpenseCategoryDTO.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package ua.ithillel.expensetracker.dto;
+
+public class ExpenseCategoryDTO {
```
Comment (ownerofglory) [inline hillel-expense-tracker-common/src/main/java/ua/ithillel/expensetracker/dto/ExpenseCategoryDTO.java]: Extend bases classes like AbstractDTO

Comment (ownerofglory) [inline hillel-expense-tracker-common/src/main/java/ua/ithillel/expensetracker/dto/ExpenseCategoryDTO.java]: Please use Lombok annotation for boilerplate code

**Review by ownerofglory — CHANGES_REQUESTED** [review]
@eIIka 
Please consider my comments:
- extend bases classes like AbstractModel or AbstractDTO
- Add class members as described in ticket (amount, category, etc.)
- Use lombok to generate boilerplate code (getters, setter, etc.)

Thanks :)

File: hillel-expense-tracker-common/src/main/java/ua/ithillel/expensetracker/dto/ExpenseCategoryDTO.java (hunk 1 above)
Comment (eIIka) [reply hillel-expense-tracker-common/src/main/java/ua/ithillel/expensetracker/dto/ExpenseCategoryDTO.java:14]: Yes, I forgot to record the changes. Fixed it

**Review by ownerofglory — APPROVED** [review]

**Pull request merged**